models/Element/AbstractElement.php

```diff
@@ -388,7 +390,7 @@ public function getVersions()
     public function __sleep()
     {
         $parentVars = parent::__sleep();
-        $blockedVars = ['dependencies'];
+        $blockedVars = ['dependencies','activeDispatchingEvents'];
```

Comment:
```suggestion
        $blockedVars = ['dependencies'];
```
This is redundant, as we have it already on `AbstractModel`.